src/lib/retire-js/RetireJsEngine.ts

```diff
@@ -159,7 +159,7 @@ export class RetireJsEngine extends AbstractRuleEngine {
 		}
 	}
 
-	// eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/require-await, no-unused-vars
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/require-await
```

Comment:
Typescript rule overrides the vanilla rule, so we only need to disable the former.